ci-operator/step-registry/ipi/conf/vsphere/ipi-conf-vsphere-commands.sh

```diff
@@ -50,6 +50,27 @@ if [ ! -z ${VERSION} ]; then
 compute:
 - name: worker
   replicas: 3
+  platform:
+    vsphere:
+      cpus: 4
+      coresPerSocket: 1
+      memoryMB: 16384
+      osDisk:
+        diskSizeGB: 120"
+    fi
+    if [[ "${SIZE_VARIANT}" == "compact" ]]; then
```

Comment:
@WenXinWei as we talked before, I think we don't need to put the `SIZE_VARIANT == compact` in the `if [ ! -z ${VERSION} ];` section, 
it should have no relationship with the VERSION parameter.